design/ceph/object-store-zonegroup.md

```diff
@@ -0,0 +1,128 @@
+# Rook Object Store Zonegroup
+
+## Prerequisites
+
+A rook ceph cluster with an object-store-realm resource created. 
```

Comment:
they should automatically generated (based on defaults) for backward compatibility and ease of configuration in the  single zone case